src/main/kotlin/elan/tweaks/thaumcraft/research/frontend/integration/ThaumcraftResearchTweaks.kt

```diff
@@ -35,7 +35,7 @@ object ThaumcraftResearchTweaks {
   lateinit var singletonInitializer: SingletonInitializer
 
   @Mod.EventHandler
-  fun onInit(event: FMLInitializationEvent) {
+  fun onInit(event: FMLPostInitializationEvent) {
```

Comment:
Moving it later on, just in case, it doesn't need to be initialized early anyways.